magento/magento2
# #23627 improved getCustomerName method for Order

### Description (*)
Improved `getCustomerName` method for `Order`.

### Fixed Issues (if relevant)
#23627 

### Manual testing scenarios (*)
- Register a new customer with prefix, first name, middle name, last name, and suffix.
- Place an order with the customer, check the customer name for the order confirmation email and customer details on order page, the prefix, middle name and suffix should be visible.

### Contribution checklist (*)
